### PR TITLE
Issue 377

### PR DESCRIPTION
**Pull request by jofeinstein**
- added coverage to requirements.txt
- added documentation for coverage and updated documentation for pylint

- added pylint config file `setup.cfg` that address some parts of [issue 387](https://github.com/ucb-rit/coldfront/issues/387)

    - certain warnings are ignored now

    - changed the line length from 100 to 80

    - added default flags to setup.cfg so the flags do not need to be retyped each run

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,19 +228,83 @@ emails will be outputted here for inspection.
...
+  pylint --output=<filename> <file_path>
   ```
 
+#### coverage
+
+- [coverage](https://coverage.readthedocs.io/en/6.3.2/) is a python module 
+that gauges the effectiveness of tests by measuring the code coverage 
+of Python programs.
+
+- To run coverage alongside the Django test suit, call coverage from the
```
Comment (matthew-li) [inline README.md:262]: "suit" -> "suite"

File: README.md [hunk 2]
```diff
@@ -228,19 +228,83 @@ emails will be outputted here for inspection.
...
+  - `--source` flag limits the code measured to the code within the specified 
+location, which can be either files or directories. To specify multiple 
+sources, comma separate the paths.
+
+  - `--omit` flag will not measure the coverage of code within files or 
+directories specified. Like the source flag above, comma separate 
+multiple files or directories to omit.
+
+  - For example, the following command will ignore all migration files and 
+  only measure code the statistics directory.
```
Comment (matthew-li) [inline README.md:283]: "in the"

File: README.md [hunk 3]
```diff
@@ -228,19 +228,83 @@ emails will be outputted here for inspection.
...
+  ```
+  coverage report
+  ```
+
+- To view the report in an annotated HTML format, run:
+  ```
+  coverage html
+  ```
+  
+    - This generates the directory `htmlconv` and writes the coverage
```
Comment (matthew-li) [inline README.md:301]: "htmlcov" here and below.

**Review by matthew-li — CHANGES_REQUESTED** [review]
This looks reasonable to me--thanks for adding the notes/updating `setup.cfg`.

I just left some comments about typos.

**Comment by jofeinstein** [comment]
made the changes

**Review by matthew-li — APPROVED** [review]

**Pull request merged**